platforms/hyperledger-fabric/configuration/roles/create/channel_artifacts/tasks/main.yaml

```diff
@@ -55,7 +55,11 @@
 - name: "Creating genesis block"
   shell: |
     cd {{ build_path }}
+    {% if network.version == '2.0' %}
     ./configtxgen -profile {{ genesis.name }} -channelID {{ channel_name }}  -outputBlock ./channel-artifacts/genesis.block
```

Comment:
If this is per channel, then we have to rethink when the genesis block is generated (or force only one channel when the network is in genesis mode)

Reply:
Genesis block is per channel only. So it should be fine as for a particular genesis block we will be passing its channel only